walkthroughs/howto-external-traffic/README.md

```diff
@@ -0,0 +1,177 @@
+# Configuring TLS with AWS Certificate Manager
```

Comment:
I think the README.md title can be changed

Reply:
Thanks for the catch.